integration-templates/cal-com-v2/nango.yaml

```diff
@@ -0,0 +1,163 @@
+integrations:
+    cal-com-v2:
+        syncs:
+            events:
+                sync_type: full
+                runs: every hour
+                endpoint: GET /events
+                output: Event
+                description: Retrieve all upcoming events per a user
```

Comment:
I always forget if it's worth setting track_deletes when sync_type:full

Reply:
It is. Thanks, updated